### PR TITLE
marqeta-kyc-list-user-results: kyc - function to get list of user kyc…

### PR DESCRIPTION
**Pull request by jbearden**
… results.

Pull Request for the Marqeta Client KYC module `userResults()` function. This function retrieves a list of User KYC results.

### CODE REVIEW

File: src/kyc.ts [hunk 1]
```diff
@@ -51,4 +64,55 @@ export class KycApi {
...
+  /*
+   * Function that takes a Marqeta User token and gets a list of the KYC
+   * results for that User.
+   */
+  async userResults(search :{
+    token ? : string,
+      count ? : number,
+      startIndex ? : number,
+      fields ? : string[],
+      sortBy ? : string,
```
Comment (drbobbeaty) [inline src/kyc.ts:77]: These last four lines have a little too much indentation... 

And why is there a space before, and after, the `?`? That seems different... what's up?

**Review by drbobbeaty — CHANGES_REQUESTED** [review]
just a few things

File: src/kyc.ts (hunk 1 above)
Comment (jbearden) [reply src/kyc.ts:77]: cut-n-paste carelessness

**Review by drbobbeaty — APPROVED** [review]
looks OK

**Pull request merged**